stock_mts_mto_rule/view/warehouse.xml

```diff
@@ -6,7 +6,7 @@
         <field name="model">stock.warehouse</field>
         <field name="inherit_id" ref="stock.view_warehouse"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='delivery_steps']" position="after">
+            <xpath expr="//field[@name='partner_id']" position="after">
```

Comment:
@umiphos Ok for this. But to be more coherent, shouldn't have also been located on Inventory > Settings with a related field on  res.config.settings ?

Reply:
I don't follow what you are saying, also, this field is only placed inside the warehouse and nowhere else.

Reply:
@umiphos Ok. I see. Believed it was set on company level :smile: